src/parser/jobs/drk/modules/ResourceSimulator.js

```diff
@@ -295,42 +292,42 @@ export default class Resources extends Module {
 		if (this._droppedTBNs > 0) {
 			this.suggestions.add(new TieredSuggestion({
 				icon: ACTIONS.THE_BLACKEST_NIGHT.icon,
-				content: <Fragment>
+				content: <Trans id="drk.resourceanalyzer.blackestnight.content">
 					One or more <ActionLink {...ACTIONS.THE_BLACKEST_NIGHT}/> applications did not fully use the shield, and thus did not generate a Dark Arts proc to allow free use of <ActionLink {...ACTIONS.EDGE_OF_SHADOW}/> or <ActionLink {...ACTIONS.FLOOD_OF_SHADOW}/>.
-				</Fragment>,
+				</Trans>,
 				tiers: SEVERITY_THE_BLACKEST_NIGHT,
 				value: this._droppedTBNs,
-				why: <Fragment>
+				why: <Trans id="drk.resourceanalyzer.blackestnight.why">
```

Comment:
Ah; this must've been where those plural rules in the json were coming from. Thanks for fixing those!